app/androidutils.h

```diff
@@ -34,6 +34,9 @@ class AndroidUtils: public QObject
     static void requirePermissions();
     static bool checkAndAcquirePermissions( const QString &permissionString );
 
+    //! Calls getEXIFAttribute from Java
+    static QString getExifInfo( const QString &filePath );
```

Comment:
What does this function really do? Shouldn't we have also configurable attribute name here?